components/form/FormItem.razor.cs

```diff
@@ -107,6 +108,31 @@ public partial class FormItem : AntDomComponentBase, IFormItem
         [Parameter]
         public FormValidationRule[] Rules { get; set; }
 
+        [Parameter]
+        public bool HasFeedback { get; set; }
+
+        [Parameter]
+        public FormValidateStatus ValidateStatus { get; set; }
+
+        [Parameter]
+        public string Help { get; set; }
+
+        private class IconProperties
```

Comment:
You can use ValueTuple

Reply:
have changed to ValueTuple. thanks for the advice.